TipBot/CommandModules/PublicCommands.cs

```diff
@@ -36,7 +37,38 @@ public class PublicCommands : ModuleBase<SocketCommandContext>
         /// <summary>Protects access to <see cref="CommandsManager"/>.</summary>
         private readonly object lockObject = new object();
 
-        [CommandWithHelp("tip", "Transfers specified amount of money to mentioned user.", "tip <user> <amount> <message>*")]
+		[CommandWithHelp("sc-state", "Inspects smartcontract state at a specific contract and variable", "sc-state string contractAddress, string variableName, string variableType")]
+		public Task SCStateAsync(string contractAddress, string variableName, string variableType)
```

Comment:
Don't use tabs, use 4 spaces.
That applies for all code changes. Please reconfigure you VS.

Reply:
Woops, using a different editor than normal